docs/io.cozy.files.md

```diff
@@ -97,6 +97,11 @@ The `io.cozy.files` doctype has [the standard `cozyMetadata`](https://docs.cozy.
     - `slug`: {string} the slug of the application that has made the upload
     - `version`: {string} the version number of this application
     - `oauthClient`: {map} if the upload was made by an OAuth Client, information about it (`id`, `name`, and `kind`)
+- `favorite`: {boolean} is this folder a favorite
+- `decorations`: {map} information on folder presentation (only for folders)
+    - `color`: {string} hexadecimal color code
+    - `icon`: {string} name of the icon in the list of cozy-ui icons
```

Comment:
It means that if we eventually change an icon name in cozy-ui, we will have to handle a data migration

Reply:
I wonder how it will be possible technically. Do you have any idea?